enterprise/api/v1/views.py

```diff
@@ -76,39 +108,36 @@ def courses(self, request, pk=None):
 
         Only courses with active course runs are returned. A course run is considered active if it is currently
         open for enrollment, or will open in the future.
```

Comment:
I'm wondering what the implications of your change are for this endpoint, and by extension what they are for the way we currently link a Catalog to the EnterpriseCustomer. We still rely on that link for the SAP integration and have a story to back the catalog export script with this endpoint instead of hitting the discovery service directly.